tests/functional/oauth_webchannel.js

```diff
@@ -27,23 +29,34 @@ define([
   var client;
   var ANIMATION_DELAY_MS = 1000;
 
+  var listenForSyncCommands = FxDesktopHelpers.listenForFxaCommands;
+  var testIsBrowserNotifiedOfSyncLogin = FxDesktopHelpers.testIsBrowserNotifiedOfLogin;
+
+
   /**
    * This suite tests the WebChannel functionality in the OAuth signin and
    * signup cases. It uses a CustomEvent "WebChannelMessageToChrome" to
    * finish OAuth flows
    */
 
-  function testIsBrowserNotifiedOfLogin(context, shouldCloseTab) {
+  function testIsBrowserNotifiedOfLogin(context, shouldCloseTab, wantKeys) {
```

Comment:
I felt gross adding another boolean to the end of this, but didn't want to bloat the PR any more than needed. This can be cleaned up in a follow up.